src/Silverback.Integration.Kafka/Messaging/Configuration/Kafka/ConfluentClientConfigProxies.cs

```diff
@@ -939,11 +923,7 @@ public string ConsumeResultFields
         ///     <br /><br />default: ''
         ///     <br />importance: high
         /// </summary>
-        public string GroupId
-        {
-            get => ConfluentConfig.GroupId;
-            set => ConfluentConfig.GroupId = string.IsNullOrWhiteSpace(value) ? GroupIdNotSet : value;
-        }
+        public abstract string GroupId { get; set; }
```

Comment:
Did you manually modify this or did you forget to commit the generator?

Reply:
Fixed: https://github.com/BEagle1984/silverback/pull/172/commits/f1c8a8ef8e2ef6fd3f04bf4738d1187d4a594633